File: src/main/Java/boj_step5/boj11718.java
import java.io.*;

public class boj11718 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        String line;
        while ((line = reader.readLine()) != null) {
            writer.append(line);
            writer.newLine();
            writer.flush();
        }
    }
}

File: src/main/Kotlin/others/boj11718.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    while (true) {
        r.readLine()?.let { value ->
            if (value.isNotBlank()) {
                println(value)
            }
        } ?: break
    }
    w.close()
}